ui/imports/Themes/Theme.qml

```diff
@@ -102,6 +102,7 @@ QtObject {
 
     property int additionalTextSize: 13
 
+    property int titleTextFontSize: 17
```

Comment:
I'm not sure about this naming on this global file. It would be something like the one above ... `secondaryAdditionalTextSize` for example?